R/performance-package.R

```diff
@@ -22,7 +22,6 @@
 #' References: Lüdecke et al. (2021) \doi{10.21105/joss.03139}
 #'
 #' @docType package
-#' @aliases performance performance-package
 #' @name performance-package
 #' @keywords internal
 "_PACKAGE"
```

Comment:
```suggestion
"_PACKAGE"

```

Reply:
I already added a new line locally but apparently git doesn't show it. I also can't commit your suggestion so I suppose it's not feasible?